### PR TITLE
M5G-242: fix tooltip placement

### PR DESCRIPTION
**Pull request by jonah-clever**
**Jira:**

M5G-242

**Overview:**

* Put truncated notice and tooltip/fontawesome in a span, so they would stay together on narrow screens

**Screenshots/GIFs:**

Screen Shot 2021-04-23 at 3 26 06 PM
^Note: It looks like the icon is not vertically centered with the text, but I believe that's because of the lowercase text. It should be vertically centered with capital letters

**Testing:**

- Manual tests:
  - [x] Chrome
  - [ ] Safari
  - [ ] IE11

**Roll Out:**

- Before merging:
  - [ ] ~~Updated docs~~
  - [x] Bumped version in `package.json`
    - New component or backward-compatible component feature change? Run `npm version minor`
- After merging:
  - [ ] ~~Deployed updated docs (`make deploy-docs`)~~
  - [ ] ~~Posted in #eng if I made a breaking change to a beta component~~

### CODE REVIEW

**Review by clcaelynn-clever — APPROVED** [review]
Looks great! You may have already done this, but can we get Nick's signoff on the icon-to-text alignment before merging?

**Comment by jonah-clever** [comment]
> Looks great! You may have already done this, but can we get Nick's signoff on the icon-to-text alignment before merging?

Confirmed!

**Pull request merged**
